src/reset.css

```diff
@@ -0,0 +1,65 @@
+/* http://meyerweb.com/eric/tools/css/reset/ 
```

Comment:
This is where I get my reset CSS from.  Been using his "v2" for a while now.  Any other recommendations are also welcome.

Reply:
I've been using this reset file for the whole cohort and I really like it. It is missing one thing though. The border-sizing: border-box setting. This allows you to set the size of the div INCLUDING the padding and border. Otherwise the those two are added to the width or height sizing. No idea why this isn't the default. Most people agree that adding this code makes style sizing more intuitive. My suggestion is to add this code to the bottom of your reset file. Josh endorses it too. 

`html {
box-sizing:border-box
}
* {
box-sizing:inherit
}`